src/miral/display_configuration.cpp

```diff
@@ -30,6 +33,36 @@ struct miral::DisplayConfiguration::Self : StaticDisplayConfig
 {
     Self(std::string const& name) : name{name} {}
     std::string const name;
+
+    void dump_config(std::function<void(std::ostream&)> const& print_template_config) override
+    {
+        std::string config_dir;
+
+        if (auto config_home = getenv("XDG_CONFIG_HOME"))
+            config_dir = config_home;
+        else if (auto home = getenv("HOME"))
+            (config_dir = home) += "/.config";
+
+        if (config_dir.empty())
+        {
+            mir::log_debug("Neither XDG_CONFIG_HOME or HOME is set");
```

Comment:
Maybe add why you log this: "can not write config file"

Reply:
Sure